hadoop-hdds/rocks-native/src/main/native/cplusplus_to_java_convert.h

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
I believe we still need to keep the original license header from RocksDB.

```suggestion

// Copyright (c) 2011-present, Facebook, Inc.  All rights reserved.
//  This source code is licensed under both the GPLv2 (found in the
//  COPYING file in the root directory) and Apache 2.0 License
//  (found in the LICENSE.Apache file in the root directory).

```

Reply:
RocksDB has apache2.0 license. So would that be sufficient?

Reply:
ok. this looks like a pick-one situation here. might be fine.

Reply:
Ref: https://issues.apache.org/jira/browse/LEGAL-303?focusedCommentId=16089073&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-16089073

Reply:
done